// Copyright 2015 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package cc.proto;

option optimize_for = LITE_RUNTIME;

message LayerTree {
  repeated int32 layers_that_should_push_properties = 1;
  optional bool in_paint_layer_contents = 2;
}
